powersimdata/design/investment/investment_costs.py

```diff
@@ -274,8 +283,11 @@ def _calculate_dc_inv_costs(grid_new, sum_results=True):
     """Calculate cost of upgrading HVDC lines.
 
     :param powersimdata.input.grid.Grid grid_new: grid instance.
-    :param bool sum_results: sum series to return total cost.
-    :return: (*pandas.Series/float*) -- cost of upgrading HVDC lines in $2015.
+    :param bool sum_results: whether to sum series to return total cost. Defaults to
+        True.
+    :return: (*pandas.Series/float*) -- cost of upgrading HVDC lines, in $USD,
+        inflation-adjusted to today. If ``sum_results``, a float is returned, otherwise
```

Comment:
Same as above regarding the hardcoded "today" year in `_calculate_single_line_cost`.